torch_xla/core/xla_model.py

```diff
@@ -30,7 +30,15 @@ def parse_xla_device(device):
 
 
 def get_xla_supported_devices(devkind=None, max_devices=None):
-  """get_xla_supported_devices (PLACEHOLDER)"""
+  """Returns a list of supported devices of a given kind.
+
+  Args:
+    devkind: If specified, one of `TPU`, `GPU` or `CPU`.
+    max_devices: The maximum number of devices to be returned of that kind.
```

Comment:
Btw a minor suggestion: I usually find it useful in PyTorch docs that we indicates what's the type of input arguments and return type. 
For example: 
```
dim (int)
dim (int, optional): Default is None
dims( list[int] or int)
name( string):
```
I find it useful myself that I can see what to pass in in 1s without reading the doc string :P  More examples can be found in https://github.com/pytorch/pytorch/blob/master/torch/_tensor_docs.py#L14.